app.js

```diff
@@ -60,10 +60,24 @@ _.each(routes, (verbs, url) => {
         if (!req.authUser) {
           next(new errors.HttpStatusError(401, 'Authentication required!'))
         }
-        // Access check
-        if (Array.isArray(def.access) &&
-          (!req.authUser.roles || !_.intersection(req.authUser.roles, def.access).length)) {
-          next(new errors.HttpStatusError(403, 'You are not allowed to perform this action!'))
+
+        if (req.authUser.roles) {
+          if (Array.isArray(def.access) &&
+            !_.intersection(req.authUser.roles, def.access).length) {
```

Comment:
can we normalize all roles to lower case when checking? I think that the roles come in rather inconsistently. For instance, my JWT carries the "administrator" role but the checks we have are on "Administrator".

Reply:
I'm all for that.  We'll just need to test it since I'm not sure other apps are being case sensitive for some reason (hopefully not)